packages/v4/patternfly-docs/content/design-guidelines/components/select/select.md

```diff
@@ -37,31 +37,33 @@ Single select lists allow users to select a single, mutually exclusive option. W
 ### Checkbox select
 Checkbox select lists allow users to select one or more options from a known list of options, using checkboxes as a selection method. You may choose to have a badge display in the toggle to show how many items have been selected. A common use case for checkbox select lists is in a toolbar filter for a table or page.
 
+### Use checkbox select when
```

Comment:
Make these "use xxxx select when" headers H4s or bold instead of H3s, cause I don't think they should be equal in hierarchy to their "parent" header